internal/pkg/utils/checkcontaint.go

```diff
@@ -0,0 +1,12 @@
+package utils
```

Comment:
File name should be array.go or something similar. You are not adding a file for just a single method.